adapter-cqld4/pom.xml

```diff
@@ -140,6 +140,48 @@
                     <followSymLinks>false</followSymLinks>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.8.8</version>
+                <executions>
+                    <execution>
```

Comment:
no phase for this execution? Is there a default, or does this remain unattached for some reason?

Reply:
This is for preparation of the `jacoco agent` as part of a prepare-agent goal and binds by default to the initialization [phase](https://maven.apache.org/ref/3.8.7/maven-core/lifecycles.html).

You will see this occur in the output of the logs for the jacoco-maven-plugin initialize, as it has to initialize the agent for in-memory pre-processing of all class files during class loading.

For example:
```
--- jacoco-maven-plugin:0.8.8:prepare-agent (prepare-agent) @ nb-annotations ---
2023-01-10T23:09:54.8116635Z [INFO] argLine set to -javaagent:/home/runner/.m2/repository/org/jacoco/org.jacoco.agent/0.8.8/org.jacoco.agent-0.8.8-runtime.jar=destfile=/home/runner/work/nosqlbench/nosqlbench/nb-annotations/target/jacoco.exec
```